modules/multi-arch-uninstalling-using-web-console.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies
+//
+// * post_installation_configuration/multiarch-tuning-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="multi-architecture-uninstalling-using-web-console_{context}"]
+= Uninstalling the Multiarch Tuning Operator by using the web console
+
+You can uninstall the Multiarch Tuning Operator by using the {product-title} web console.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` permissions.
+* You deleted the `ClusterPodPlacementConfig` custom resource (CR).
```

Comment:
```suggestion
* You deleted the `ClusterPodPlacementConfig` object `cluster`.
```

Reply:
I think "You deleted the `ClusterPodPlacementConfig` object" will be fine because we can only create one object  and the name must be cluster.